api/tests/integration/migrations/utils.js

```diff
@@ -199,7 +199,7 @@ const _resetDb = () => {
       logger.error('Could not create "medic-test" directly after deleting, pausing and trying again');
       return new Promise(function(resolve, reject) {
         setTimeout(function() {
-          db.get('medic-test')
+          db.exists('medic-test')
```

Comment:
I don't think this is correct - it's meant to be trying to create it again. `db.exists` explicitly does not create the db.

Reply:
Sorry you're right!